crates/bevy_app/src/app.rs

```diff
@@ -63,6 +63,12 @@ impl App {
     }
 
     pub fn run(mut self) {
+        // Setup the default bevy task pools
+        self.resources
```

Comment:
I'm not convinced bevy_tasks should be hard-coded into apps, especially given that we're just adding resources. Can we make this a plugin that we add with `add_default_plugins()`?

Reply:
Making a plugin in bevy_tasks will cause circular dependencies (app depends on ecs, ecstatics depends on tasks). Where to do you want it to be?